portfolio/src/main/webapp/index.html

```diff
@@ -53,6 +53,8 @@ <h1>Elvis Vasquez</h1>
         <form action="/data" method="POST">
```

Comment:
I think this action is not supposed to be hardcoded since it's set by fetchBlobstoreUrl

https://happycoding.io/tutorials/google-cloud/blobstore

Reply:
You're the action will be replaced by the URL, so there's no point in having it there